config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -879,3 +879,33 @@ periodics:
     testgrid-dashboards: sig-node-containerd
     testgrid-tab-name: cgroup-systemd-containerd-node-e2e
     testgrid-alert-email: kubernetes-sig-node-test-failures@googlegroups.com
+- interval: 12h
+  name: ci-cos-cgroupv2-containerd-e2e-gce
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-e2e-cos-containerd: "true"
+  spec:
+    containers:
+    - args:
+      - --repo=github.com/containerd/containerd=master
+      - --timeout=70
+      - --scenario=kubernetes_e2e
+      - --
+      - --check-leaked-resources
+      - --env=ENABLE_POD_SECURITY_POLICY=true
+      - --env=KUBE_NODE_EXTRA_METADATA=user-data=/workspace/github.com/containerd/containerd/test/e2e/node.yaml,containerd-configure-sh=/workspace/github.com/containerd/containerd/cluster/gce/configure.sh,containerd-env=/workspace/test-infra/jobs/e2e_node/containerd/containerd-master/cgroupv2/env-cgroupv2
+      - --extract=ci/latest
+      - --gcp-node-image=gci
+      - --gcp-nodes=4
+      - --gcp-zone=us-west1-b
+      - --ginkgo-parallel=30
+      - --image-family=cos-beta
```

Comment:
Maybe leave comment here why we using `cos-beta`, similar as https://github.com/kubernetes/test-infra/pull/23721/files

Reply:
Thanks, added the comment for using `cos-beta`.